netbox/dcim/tables/devices.py

```diff
@@ -359,6 +359,17 @@ class CableTerminationTable(NetBoxTable):
         verbose_name=_('Mark Connected'),
     )
 
+    def value_link_peer(self, value):
+        string = ""
+        for termination in value:
+            if value and not str:
+                string += " "
+            if termination.parent_object:
+                string += f"{termination.parent_object} > "
+            string += str(termination)
+
+        return string
```

Comment:
We can condense this (it's safe to assume every termination object has a `parent_object` attribute).
```suggestion
        return ', '.join([
            f"{termination.parent_object} > {termination}" for termination in value
        ])
```